example/lib/main.dart

```diff
@@ -340,6 +340,18 @@ class _MyBodyState extends State<Body> {
                 }
               },
             ),
+            ElevatedButton(
+              child: const Text("Send Stream Payload"),
+              onPressed: () async {
+                endpointMap.forEach((key, value) {
+                  String a = Random().nextInt(100).toString();
+
+                  showSnackbar("Sending $a to ${value.endpointName}, id: $key");
+                  Nearby()
+                      .sendStreamPayload(key, Uint8List.fromList(a.codeUnits));
+                });
```

Comment:
This is not really a usecase for streams, nor it depicts how to send an endless stream of data.

See https://github.com/googlearchive/android-nearby/blob/master/connections/walkietalkie/app/src/automatic/java/com/google/location/nearby/apps/walkietalkie/MainActivity.java

the linked example uses audio, but if we want to avoid that, we can instead ask the user to hold the button, and send random bytes of payload while the button is on hold. Basically an example of sending a stream of data without any known size

Reply:
@mannprerak2 Bro, have you tested the streams in Java code yourself? Because, the streaming only works for the first time we use Nearby.getConnectionsClient(activity).sendPayload(endpointId, Payload.fromStream(inputStream)). If I do the same for the second time, the streaming doesn't work. Here is the code which I've written for the logic. https://github.com/lakshmanprabhu49/nearby_connections/tree/Streaming_v2

Reply:
You can push commits to this PR itself.

Reply:
@mannprerak2 I've pushed the code here. Can you review? As far as I tested in my local, the stream was sent from sender to receiver. But not sure if this will work in all scenarios

Reply:
I've added more comments

Reply:
Could you also change the example to send maybe a stream of numbers at a gap of 1 second. E.g -
```dart
Stream<int>.periodic(const Duration(seconds: 1), (x) => x).take(5)
      .listen((i) {
        Nearby().sendStreamPayload(key, Uint8List.fromList(i.toString().codeUnits));
      });

```